bin/generate_dist

```diff
@@ -54,9 +54,22 @@ def generate_dist_files(locale)
     file.write(JSON.pretty_generate(values_json))
     file.write("\n")
   end
-  @cli.write_file!("dist/#{locale}/mappings.json") do |file|
-    file.write(JSON.pretty_generate(mappings_json))
-    file.write("\n")
+  if locale == "en"
```

Comment:
Why did we do only english here? I think we should have the mappings in every version of the distribution. 🤔